api/illumio/cloud/k8scluster/v1/resource_config.proto

```diff
@@ -94,6 +95,185 @@ message KubernetesResourceMutation {
   }
 }
 
+// Information identifying a Cilium flow.
```

Comment:
1. This doesn't just _identify_ a flow, it _is_ a flow. Rephrase that in the comment for each message below.
2. Refer to the Observer service definition.
```suggestion
// A flow received from Cilium Hubble Relay using the Observer service's GetFlows RPC.
// https://github.com/cilium/cilium/blob/main/api/v1/observer/observer.proto
```

Reply:
Done